### ISSUE TITLE
No Automation possible for On/Off State?

### ISSUE THREAD

**Issue by drheck** [QuickSander/homebridge-mieleathome#74]
Hi QuickSander,
i have a dishwasher and there is only one device with to devices combined in homekit (Temperaturesensor and an Dishwasher element)
But i can only add an automation for the temperaturesensor: if temp > 0 then it seems t be on and if temp < 1 it is off. But this does nothing, because the temperature sensor has no actual value.

It would be very nice to add an automation to the Dishwasher element in order to trigger start/stop events. 
What is wrong?

Thanks a lot,
Peter

**Comment by EdoHamburg** [comment]
Yeah, I'd like to start the preprogrammed washing machine and dishwasher, when my energy price is low (I got a “light sensor” in my HomeKit, indicating a low price). But I can't add the valve for any automation in HomeKit. Any workaround available?

Edo
